src/trackinfoobject.h

```diff
@@ -307,6 +325,14 @@ class TrackInfoObject : public QObject {
     // TrackDAO
     void setId(TrackId id);
 
+    // Stores a pointer to the original track for temporary instances that
+    // have been created by cloning another track with cloneTemporary().
+    // Ensures that the lifetime of cloned instances does not exceed the
+    // lifetime of their parent tracks. Especially important for parent
+    // tracks that are managed by TrackCache, because tags might be written
+    // back into the file when the track is evicted from the cache.
+    const TrackPointer m_pParentTrack;
```

Comment:
I have still not understand why this is required and why we have a problem. 
When the parent track falls out of scope and is deleted, it was not needed any more. Why is it require to keep it alive?